requirements-data_and_models.txt

```diff
@@ -15,6 +15,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+cupy-cuda102==8.6.0
```

Comment:
From what I understand, yes it is. It is the only `added` library when you `pip install spacy[cuda102]` instead of `pip install spacy`. 

I agree, it is a problem! To be honest, I don't know what is the best solution for this! Should I comment this line out ?